entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eu # Increase bash strictness
+
+if [[ -n "${GITHUB_WORKSPACE}" ]]; then
+  cd "${GITHUB_WORKSPACE}/${INPUT_WORKDIR}" || exit
+fi
+
+export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
+
+export REVIEWDOG_VERSION=v0.11.0-nightly20210104+207b245
+
+echo "[action-flake8] Installing reviewdog..."
+wget -O - -q https://raw.githubusercontent.com/reviewdog/nightly/master/install.sh | sh -s -- -b /tmp/ "${REVIEWDOG_VERSION}"
```

Comment:
suggestion:
After main process done, don't you clean up reviewdog?

(nothing ok)

Reply:
Very good point!

Reply:
@tsuyoshicho I now added it in the entrypoint.sh file. When composite actions mature more and also support the use and if statements our code can be cleaned up (see https://github.com/actions/runner/issues/646).

https://github.com/reviewdog/action-flake8/blob/811d99ae63a3cd3ab7226eb4f96be43256aff0fa/entrypoint.sh#L33-L38